styles.css

```diff
@@ -128,3 +128,333 @@ a:hover {
 .socials a:active {
   fill: blue;
 }
+
+/* end of page 1 styling */
+
+/* card 1 styling */
+.work-container {
+  /* border: 2px solid red; */
+  border: 1px solid #cbcdd1;
+  display: grid;
+  grid-template-columns: auto;
+  justify-content: center;
+  align-content: center;
+  text-align: left;
+  width: 95%;
+
+  /* margin-top: 910px; */
+  margin-top: 110vh;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 50px;
+  gap: 12px;
+  min-height: 552px;
+  background: #fff;
+  border-radius: 16px;
+}
+
+.portfolio-img {
+  /* border: 2px solid brown; */
+  width: 339px;
+  height: 220px;
+  border-radius: 8px;
+  margin-top: -9%;
+}
```

Comment:
- [ ] I have noticed that you have the commented codes in the CSS file. Kindly ensure that you follow [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). There fore I suggest that the commented codes are removed to make your code clean and more professional ☺️ .